zero/gkyl_rect_grid.h

```diff
@@ -29,6 +29,20 @@ struct gkyl_rect_grid {
 void gkyl_rect_grid_init(struct gkyl_rect_grid *grid, int ndim,
   const double *lower, const double *upper, const int *cells);
 
+/**
+ * Find cell indices of point
+ *
+ * @param grid Grid object
+ * @param point The point to find the cell indices at
+ * @param pick_lower
```

Comment:
@jRoeltgen can you add a description for this argument please?

Reply:
Added "If point on cell boundary, pick lower cell if true, and upper if false"